src/css/custom.css

```diff
@@ -127,7 +127,7 @@
 }
 
 .text-slate\/60 {
-  color: rgb(67 84 101 / 0.6);
+  color: var(--ifm-footer-color);
```

Comment:
What does this look like for you?

Reply:
![SCR-20231219-piak](https://github.com/pomerium/documentation/assets/60552605/83ebf920-6d25-4729-a8b8-5a12e47bb55d)

Reply:
The button could use a `{cursor: pointer}` rule as well so the mouse switches from the arrow to the glove.